src/pocketmine/plugin/PluginDescription.php

```diff
@@ -170,6 +174,24 @@ public function getCompatibleApis() : array{
 	public function getCompatibleMcpeProtocols() : array{
 		return $this->compatibleMcpeProtocols;
 	}
+	
+	/**
+	 * @return string[]
+	 */
+	public function getCompatibleOperatingSystems() : array{
+		return $this->compatibleOperatingSystems;
+	}
+	
+	/**
+	 * Checks if the current operating system is in the compatible systems list.
+	 *
+	 * @throws PluginException if the current operating system is not compatible
+	 */
+	public function checkCompatibleOperatingSystem() {
```

Comment:
I think this function is redundant: it's only used in 1 place and it can be checked in the same fashion as everything else [here](https://github.com/pmmp/PocketMine-MP/pull/3192/files#diff-10c30a902a542d91471a1a004b245c13R256-R286).